### PR TITLE
Improve PowerPC64 Makefile support

### PR DESCRIPTION
**Pull request by antonblanchard**
Remove Make.powerpc, and integrate support into the main Makefiles.

### CODE REVIEW

File: Make.inc [hunk 1]
```diff
@@ -606,6 +606,17 @@ else
...
 endif
 
+# If we are running on powerpc64, set certain options automatically
+ifneq (,$(findstring powerpc64,$(ARCH)))
+JCFLAGS += -fsigned-char
+
+override LLVM_VER:=3.8.0
+override OPENBLAS_DYNAMIC_ARCH:=0
+override OPENBLAS_TARGET_ARCH:=POWER8
+override USE_SYSTEM_LIBM:=1
```
Comment (Keno) [inline Make.inc]: Does openlibm not work on Power?

Comment (antonblanchard) [inline Make.inc]: It does, but Julia pulls in an older version. Could we update the version Julia uses?

Comment (Keno) [inline Make.inc]: Yes, we need to do that anyway. cc @ViralBShah

Comment (tkelman) [inline Make.inc]: #16421

**Comment by antonblanchard** [comment]
Thanks for updating openlibm. I've rebased my patch to enable openlibm on PowerPC64.

As a bonus, it has cleared up a number of testcase failures.

**Comment by Keno** [comment]
LGTM.

**Comment by Keno** [comment]
Somebody remind me why we're setting fsigned-char here?

**Comment by ViralBShah** [comment]
There is an issue about this and the compiler has this assumption somewhere. We had to first do it on arm.

**Pull request merged**